cypress/e2e/tables.cy.js

```diff
@@ -0,0 +1,14 @@
+import {User} from "@nextcloud/cypress"
+describe('The Home Page', () => {
+	it('successfully loads', () => {
+		const user = new User('admin', 'admin')
```

Comment:
In order to make sure that the tests run isolated it would be good to create a fresh user for the test. 

There is a helper function for this https://nextcloud.github.io/nextcloud-cypress/functions/commands.createRandomUser.html

An example of this being used could be https://github.com/nextcloud/viewer/blob/b2553ddf080754cb52549ecd569568f98d42ae7c/cypress/e2e/actions/delete.cy.js#L26-L33